src/components/Accordion.js

```diff
@@ -18,6 +18,17 @@ const Accordion = React.createClass({
       created: [] };
   },
 
+  componentWillReceiveProps: function(nextProps) {
+    const opened = [...this.state.opened];
```

Comment:
it's being used in handleHeaderClick down below, seems like copying to a new array, although  unnecessary here cause I use .map (realizing  just now :)  )